spec/controllers/v3/tags_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V3::TagsController do
+  subject { create :repository_compound }
```

Comment:
Actually, the subject of the test is the `TagsController` instead of a repository, so please make this a `let!(:repository)` definition.